app/services/cart.js

```diff
@@ -216,10 +227,20 @@ export default ApiService.extend({
   },
 
   remove(pkgOrItem) {
+    let promise;
+    let loadingView = getOwner(this)
```

Comment:
We can use `runTask` from `asyn_task` mixins.

Reply:
Eg: https://github.com/crossroads/browse.goodcity/blob/master/app/controllers/order/confirm_booking.js#L16